semcore/notice/src/Notice.jsx

```diff
@@ -154,6 +165,8 @@ function Close({ styles, getI18nText }) {
 
 export default createComponent(RootNotice, {
   Label,
+  Title,
+  Text: NoticeText,
```

Comment:
Let's use here Text and in import `{ Text as TypographyText}` or something like that